record-and-playback/core/lib/recordandplayback/edl/audio.rb

```
@@ -234,6 +301,169 @@ def self.process_segment(entry, output_basename, i, total_segments, audioinfo)
+            last_interval = merged_recode_intervals.last
+            if current_interval[:start] <= last_interval[:end] + 0.01
+              last_interval[:end] = [last_interval[:end], current_interval[:end]].max
+            else
+              merged_recode_intervals << current_interval
+            end
+          end
+        end
+
+        segments = []
+        last_processed_pts = 0.0
+        
```

RuboCop (1.82.1)

[convention] Trailing whitespace detected.

(Layout/TrailingWhitespace)